brainiak/funcalign/srm.py

```diff
@@ -552,6 +552,87 @@ def _srm(self, data):
         sigma_s = self.comm.bcast(sigma_s)
         return sigma_s, w, mu, rho2, shared_response
 
+    def score(self, data):
+        """Calculate the log-likelihood of test-subject's data for the model selection
+
+        Parameters
+        ----------
+
+        data : list of 2D arrays, element i has shape=[voxels_i, samples]
+            Each element in the list contains the fMRI data of one subject.
+
+
+        Returns
+        -------
+
+        ll_score : Log-likelihood of test-subject's data
+        """
```

Comment:
Might be better to have an input validation here to make sure all `d` in `data` have the same number of samples? 

I noticed that later on line 616, you are skipping any `d` in `data` that is `None`. I was wondering if this can be done earlier, such a removing `None` from the very beginning, or warn the user that there is None in the `data` list? Feel free to decide what's more user-friendly.

Reply:
ah and this function probably want to assert `len(data) >= 2`